### DISCUSSION TITLE
css Not loading

### DISCUSSION THREAD

**Discussion by nothikarulol** [vercel/community#5063] [Help] [answered]
I tried to upload my site to vercel, But I deployed it and all it shows is the html and not CSS, anything I can do to fix it?

**Comment by leerob** [comment]
Hey! Could you have the URL for the site as well as the code possibly?

**Reply by nothikarulol** [reply]
You want me link the github for you so you can check it out?

**Reply by nothikarulol** [reply]
here is the website: https://adamsgameshack.vercel.app/ and the repository is: https://github.com/nothikarulol/Adamsgameshack

**Reply by amyegan** [reply]
Hi @nothikarulol! It looks like you made some changes to the code since your last comment. The new version I pulled seems to be working. Are you still having any trouble with static files not loading?

**Reply by yuelinwen** [reply]
Hi, @amyegan I have same issue, the CSS is not loading in vercel, do you mind to check for me? 
my github repo:https://github.com/yuelin-wen/cyber-comics-prj
my vercel: https://cyber-comics-prj.vercel.app/2746

thank you!

**Reply by nothikarulol** [reply]
Try renaming your css file style.css if its styles.css.

**Reply by BrendoLima** [reply]
Interestingly, this solution worked for me, it must have to do with the Versel builder.

**Reply by Vikin-5352** [reply]
This solved my issue

**Reply by Yashesh133** [reply]
Hi, @amyegan I have same issue, the CSS is not loading in vercel, do you mind to check for me?
my github repo:https://github.com/Yashesh133/Travel-Website-UI-UX-Design
my vercel: https://travel-website-ui-ux-design.vercel.app/

**Reply by GeethuGeorge** [reply]
@Yashesh133  Hi Yashesh, i haved the same issue, css not loading properly in responsiveness, how did you get it corrected?

**Reply by priyansane** [reply]
change the folder structure there should be one src folder with index.html and inside of src have your all the necessary folder worked for me.

**Reply by nazish-16** [reply]
@leerob @amyegan i have been facing this issue too, can you guys help me too
here's my github repo: https://github.com/nazish-16/sayeedia-gallery
vercel website: https://sayeedia-gallery.vercel.app/

**Comment by nothikarulol** [comment]
No i got it fixed but thanks for trying!

**Reply by yuelinwen** [reply]
hi mate, may i ask what things you've modified? seems like I have the same issue. thank you~

**Comment by Yashesh133** [comment]
I have same issue, the CSS is not loading in vercel, do you mind to check for me?
my github repo:https://github.com/Yashesh133/Travel-Website-UI-UX-Design
my vercel: https://travel-website-ui-ux-design.vercel.app/

**Reply by Aryan-yadav15** [reply]
how did you fix it?

**Reply by Yashesh133** [reply]
Check the file structure, make the folder structure using the src directory

**Comment by yuelinwen** [comment, marked as answer]
Check your root file path on vercel, maybe that’s the issue. Vercel can’t
 find your css file properly

Aryan Yadav ***@***.***>于2024年1月7日 周日上午3:03写道：

**Comment by simonh1000** [comment]
I'm seeing the same but only on certain devices -1 apple, 1 android. its really mysterious

**Comment by MuhammadAhmadSaaim** [comment]
Can someone check what's wrong with my app too?
Vercel Link: https://muhammadahmadsaaim.vercel.app/
Github Link: https://github.com/MuhammadAhmadSaaim/Portfolio

**Reply by Yashesh133** [reply]
Check the file structure, make the folder structure using the src directory

**Comment by Shanu2604** [comment]
I am having css and app functionality related problems.... Can anyone help me?
Github- https://github.com/Shanu2604/TVFLIX
Vercel - https://tvflixwebapp.vercel.app/

**Reply by Yashesh133** [reply]
Check the file structure, make the folder structure using the src directory

**Reply by haydenfaw** [reply]
@Yashesh133 can you explain what you mean by that? I cannot get this to work on mine either and I am all turned around at this point.

**Comment by adighiero1** [comment]
can someone help me with this? I am clueless I am having the same issue and I can't seem to figure it out
https://github.com/adighiero1/links
https://links-eight-nu.vercel.app/ the CSS and the pictures are not rendering.
